internal/web/middlewares/authentication.go

```diff
@@ -158,12 +158,22 @@ func (config *JwtConfig) ValidateToken(auth string) (*jwt.Token, error) {
 	audience := config.Audience
 	issuer := config.Issuer
 
-	checkAud := claims.VerifyAudience(audience, false)
+	checkAud := false
+	for _, aud := range audience {
+		if claims.VerifyAudience(aud, false) {
+			checkAud = true
```

Comment:
need to break out of loop on success? same in l174

Reply:
Not necessary as it's not set to false other than during initialization.